shinydashboard/lantern/server.R

```diff
@@ -1,5 +1,19 @@
 # Define server function
 function(input, output, session) {
+  observeEvent(session, {
```

Comment:
is this just called once on page load? i'm just trying to understand the difference between these two observeEvents; if you could add a comment that would be super helpful!

Reply:
I added some comments to the observeEvent functions to make this more clear!